docs/source/install-tutorial.rst

```diff
@@ -30,24 +45,27 @@ commands to run
   . bin/activate
   cd repo
   # install bepasty and requirements
-  pip install -e .
+  pip3 install -e .
   # add gunicorn and gevent for hosting
-  pip install gunicorn gevent
+  pip3 install gunicorn gevent
 
-config file for bepasty -- ``/home/bepasty/bepasty.conf``:
+Config file for bepasty -- ``/home/bepasty/bepasty.conf``:
 
 Copy ``src/bepasty/config.py`` to ``/home/bepasty/bepasty.conf`` first,
 remove the ``class Config`` and remove all indents in the file.
-The comments can be removed too, if you feel the need to.
-At last modify these two configs variables:
+The comments can be removed too, if you feel the need to.  If you wish 
+to use a script to do this in one go - please see the Tips & Tricks below.
+
+At least modify these four config variables.  
 
 ::
 
+  SITENAME = 'bepasty.example.org'
   STORAGE = 'filesystem'
   STORAGE_FILESYSTEM_DIRECTORY = '/home/bepasty/storage/'
+  SESSION_COOKIE_SECURE = False
 
-
-add this content to ``/home/bepasty/bin/gunicorn_bepasty``:
+Add this content to ``/home/bepasty/bin/gunicorn_bepasty``:
```

Comment:
you added `.local/bin` to PATH above, but here you use `bin`.

Reply:
Don't quite follow?  
If you are referring to /home/bepasty/bin/gunicorn_bepasty - this came from the previous instructions. I didn't change anything here and gunicorn seems to expect the config there.

Reply:
ah, yeah, ignore my comment. this `bin` comes from the virtualenv.